files/templates/submission.html

```diff
@@ -512,7 +512,7 @@ <h5 class="card-title">Jump in the discussion.</h5>
 	</script>
 {% endif %}
 
-{% elif not p.replies and p.deleted_utc == 0 %}
+{% elif not p.replies and p.state_user_deleted_utc == None %}
```

Comment:
I think it's actually canonically `none` in Jinja (though they might have aliased `None` in recent versions).
Jinja also has a template test for none-ness: `p.state_user_deleted_utc is none`

Reply:
Should now be fixed.